src/ignitus-Dashboard/ignitus-Dashboard-Header/Components/dashBoardHeader.js

```diff
@@ -4,20 +4,33 @@
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 import React from 'react';
-import { Redirect, Link } from 'react-router-dom';
+import '../Styles/style.scss';
+import ProfileDropDown from '../Components/profileDropdown';
+import { Redirect} from 'react-router-dom';
 import { HashLink } from 'react-router-hash-link';
 import logo from '../../../ignitus-Assets/Images/nav-logo.svg';
 import { withErrorBoundary } from '../../../ignitus-Shared/Components/errorBoundary';
+import StatusDropdown from './statusDropdown';
 
 class dashBoardHeader extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
```

Comment:
State declaration can be done using react `useState hook` => https://reactjs.org/docs/hooks-state.html